fpga_interchange/populate_chip_info.py

```diff
@@ -208,12 +210,19 @@ def __init__(self, device, tile_type_index, tile_type, cell_bel_mapper,
                 site_index=None)
             self.add_wire(flat_wire)
 
-        # Add pips
+        # Add pips, collecting pseudo_pips for later processing.
+        pseudo_pips = []
         for idx, pip in enumerate(tile_type.pips):
-            # TODO: Handle pseudoCells
-            self.add_tile_pip(idx, pip.wire0, pip.wire1)
+            pip_index = self.add_tile_pip(idx, pip.wire0, pip.wire1)
+
+            is_pseudo_cell = pip.which() == 'pseudoCells'
```

Comment:
What is a pseudoCells here? I see that it is in the fpga-schema, and what I don't fully get is why there are multiple pseudoCells for one pip.

Reply:
https://github.com/SymbiFlow/fpga-interchange-schema/pull/18